src/rest.cpp

```diff
@@ -519,7 +519,6 @@ static bool rest_getutxos(HTTPRequest* req, const std::string& strURIPart)
             bitmapStringRepresentation.append(hit ? "1" : "0"); // form a binary string representation (human-readable for json output)
         }
     }
-    boost::to_block_range(hits, std::back_inserter(bitmap));
```

Comment:
Removing this, we need to manually update the bitmap somewhere, otherwise it remains always empty.
E.g. can add  this in the last loop over `vOutPoints`:
```cpp
bitmap[i / 8] |= ((uint8_t)hit) << (i % 8);
```

Reply:
yep, just missed this. good catch